packages/quantic/force-app/main/default/lwc/quanticPager/e2e/pagerObject.ts

```diff
@@ -0,0 +1,72 @@
+import {Page} from '@playwright/test';
+import {isUaCustomEvent} from '../../../../../../playwright/utils/requests';
+
+export class PagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
```

Comment:
If you want to expose it?

```suggestion
export class PagerObject {

  public|private page: Page;

  constructor(public page: Page) {
    this.page = page;
  }
```

Reply:
Unless we feel the need to expose I wouldn't,